tensorflow/core/kernels/data/dataset_test_base.cc

```diff
@@ -284,6 +286,65 @@ Status DatasetOpsTestBase::RunOpKernel(OpKernel* op_kernel,
   return context->status();
 }
 
+Status GetOpSig(const string& op, const OpDef** sig) {
```

Comment:
can you just pass this as a lambda into `InstantiateFunction`?

Reply:
Yes, will do.